client/src/components/layouts/global.css

```diff
@@ -519,6 +519,19 @@ form {
 input:focus {
   border-color: var(--tertiary-color);
 }
+
+/*
+ * Text within backticks in en-US, es, and zh-CN challenge files
+ * is parsed into spans with this class.
+ */
+.highlighted-text {
+  padding: 1px 4px;
+  background-color: var(--tertiary-background);
+  color: var(--tertiary-color);
+  border: 1px solid var(--gray-45);
+  font-size: 90%;
+}
```

Comment:
Cloned the `code` styles, with `border-radius` and `overflow-wrap` rules excluded:
- `border-radius` should be 0 here by default
- `overflow-wrap: anywhere` doesn't make sense for language text

https://github.com/freeCodeCamp/freeCodeCamp/blob/a7f67e349c0440f31cb4e8c0b34e2c4f36f7f775/client/src/components/layouts/global.css#L522-L530